### PR TITLE
Update audio manipulation tutorial 

### PR DESCRIPTION
**Pull request by carolineechen**
part of torchaudio 0.9 release [checklist](https://github.com/pytorch/audio/issues/1527)

- update HTK support and metadata printing
- add resampling tutorial

### CODE REVIEW

File: beginner_source/audio_preprocessing_tutorial.py [hunk 1]
```diff
@@ -52,6 +52,8 @@
 import requests
 import matplotlib
 import matplotlib.pyplot as plt
+import numpy as np
```
Comment (mthrok) [inline beginner_source/audio_preprocessing_tutorial.py:55]: Is this used?

Comment (carolineechen) [reply beginner_source/audio_preprocessing_tutorial.py:55]: this is needed for comparison to librosa, which takes an np array as input

Comment (mthrok) [reply beginner_source/audio_preprocessing_tutorial.py:55]: But I do not see any use of `np.XX`.

Comment (carolineechen) [reply beginner_source/audio_preprocessing_tutorial.py:55]: you're right, importing np is not necessary

**Pull request merged**